mantle/cmd/ore/aliyun/tag-image.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2017 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aliyun
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	cmdTagImage = &cobra.Command{
+		Use:   "tag-image --id <id> --tags foo=bar ...",
+		Short: "Tag an image",
+		Run:   runTagImage,
+	}
+	id     string
+	tags   []string
+	region string
+)
+
+func init() {
+	// Initialize the command and its flags
+	Aliyun.AddCommand(cmdTagImage)
+	cmdTagImage.Flags().StringVar(&id, "id", "", "Aliyun Image ID")
+	cmdTagImage.Flags().StringVar(&region, "region", "", "Region")
+	cmdTagImage.Flags().StringSliceVar(&tags, "tags", []string{}, "list of key=value tags to attach to the Aliyun image")
```

Comment:
Would be good to add `--tags` to `ore aliyun create-image` too for consistency.

Similarly for GCP and Azure.

For cost reporting purposes, I would expect us actually to want to set the tags on upload instead of as a postprocess step.